src/styles/block/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+
+  &__list {
+    width: $widthCalendar;
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap;
+
+    @each $weekday, $number in $days {
+      &--start-day--#{$weekday} {
+        & :first-child {
+          margin-left: $number * ($ceilSize + $gap + 2px);
```

Comment:
I think 2 px isn't clear and looks like a magic digit. I don't understand what it means